src/redux/books/BookSlice.js

```diff
@@ -0,0 +1,15 @@
+export const addBook = 'ADD_BOOK';
+export const removeBook = 'REMOVE_BOOK';
+
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case addBook:
+      return state.concat(action.payload);
+    case removeBook:
+      return state.map((book) => book.id !== action.payload);
```

Comment:
- [ ] Kindly use `filter` instead of map:
`return state.filter((book) => book.d !== action.payload);`